### PR TITLE
Bug 1162886 - remove things that have been removed from the add-on

### PR DESCRIPTION
**Pull request by fmarier**
https://bugzilla.mozilla.org/show_bug.cgi?id=1162886

### CODE REVIEW

**Comment by flodolo** [comment]
Since this page is localized (in 28 locales), unfortunately you need a bit more code.

Basically, we add changed strings (1st and 3rd in your current PR) behind a tag: if the locale has the tag, it displays the new string, otherwise it displays the old one. This way we don't display English text in localized pages, and en-US is free to go without waiting for localization.

See for example this [piece of code](https://github.com/mozilla/bedrock/blob/7a85e99c87e069007fa434853b2e7981b0a5ba22/bedrock/mozorg/templates/mozorg/about/history.html#L103-L111).

Side note: since localizers are already busy with spring release material, I won't expose this update for a bit to avoid creating unnecessary noise.

**Comment by fmarier** [comment]
> Basically, we add changed strings (1st and 3rd in your current PR) behind a tag: if the locale has the tag, it displays the new string, otherwise it displays the old one.

So what you're asking is for me to add an `if l10n_has_tag('update_may2015')` statement and put the new strings in there?

> Side note: since localizers are already busy with spring release material, I won't expose this update for a bit to avoid creating unnecessary noise.

No worries, it's not urgent.

**Comment by flodolo** [comment]
Something like 
```
{% if l10n_has_tag('update_may2015) %}
    {{ _('This is the new string') }}
{% else %}
    {{ _('This is the old string') }}
{% endif %}
```
Only for the string that actually changed. Obviously no need when you're removing a string.

**Comment by fmarier** [comment]
I've re-pushed with a new tag.

File: bedrock/lightbeam/templates/lightbeam/about.html [hunk 1]
```diff
@@ -101,7 +105,11 @@ <h1 class="page-slogan">{{_('About Lightbeam')}}</h1>
   <div class="span6">
     <h2>{{_('Data formats in Lightbeam')}}</h2>
     <p>
+{% if l10n_has_tag('update_may2015) %}
+      {{_('Visit our <a href="%s">documentation on Github</a> to learn more about the file formats used in Lightbeam.')|format('https://github.com/mozilla/lightbeam/blob/master/doc/data_format.v1.2.md')}}
+{% else %}
       {{_('Visit our <a href="%s">documentation on Github</a> to learn more about the file formats used in Lightbeam and to see details of the data that is uploaded if you opt in to contribute data.')|format('https://github.com/mozilla/lightbeam/blob/master/doc/data_format.v1.1.md')}}
```
Comment (flodolo) [inline bedrock/lightbeam/templates/lightbeam/about.html]: I think you can use the updated URL here, even if the string is the old one.

**Comment by fmarier** [comment]
Done. Now using the new URL in both cases.

**Comment by AssholeDotzler** [comment]
Usted ama el dolor? Y para que te haga un favor... voy a hacer eso!


Yo saque sus huevos de oro de fracaso tan duro hasta que están sonando y que están cantando todas las melodías de dolor hacia adelante y hacia atrás en un instante! Yo soy el cascanueces chico que las grietas de su nueces que nutheads!

Y ahora vete a la mierda ESTÚPIDA de MIERDA COÑO!

----

Usted puede lamer mi Pene y mis Huevos de oro y besar mi Culo hijo de puta con un Trastorno de la personalidad desde el más profundo de los abismos del infierno desde los Abismos de Mozilla!

Vete a la mierda y que tu Madre te autista hijo de puta, vas a lamer Putin Huevos, Puto Maricon Fascista!

La mierda de su Perro, su Gato y su sucio Cabra de montaña que Maricon Pendejo!

Chupar Mozilla Pollas y todos juntos ir y chupar de Google Culo lleno de Mierda!

A la mierda en su rodilla le cock suckers! Y que Carajo Australis Chrome clon de mierda! Follando Mozilla gay lesbiana homosexual hijos de puta del infierno!

File: bedrock/lightbeam/templates/lightbeam/about.html [hunk 2]
```diff
@@ -45,10 +45,14 @@ <h1 class="page-slogan">{{_('About Lightbeam')}}</h1>
       {{_('Lightbeam visualizes the relationships between the sites you visit and the third party sites that are active on those pages.')}}
     </p>
     <p>
+{% if l10n_has_tag('update_may2015) %}
```
Comment (alexgibson) [inline bedrock/lightbeam/templates/lightbeam/about.html]: You're missing a closing `'` here in the tag name. It should be `{% if l10n_has_tag('update_may2015') %}`

While you're updating, if you could also indent these tags to sit inside the `<p>` that would be great. Thanks

File: bedrock/lightbeam/templates/lightbeam/about.html [hunk 3]
```diff
@@ -101,7 +105,11 @@ <h1 class="page-slogan">{{_('About Lightbeam')}}</h1>
   <div class="span6">
     <h2>{{_('Data formats in Lightbeam')}}</h2>
     <p>
-      {{_('Visit our <a href="%s">documentation on Github</a> to learn more about the file formats used in Lightbeam and to see details of the data that is uploaded if you opt in to contribute data.')|format('https://github.com/mozilla/lightbeam/blob/master/doc/data_format.v1.1.md')}}
+{% if l10n_has_tag('update_may2015) %}
```
Comment (alexgibson) [inline bedrock/lightbeam/templates/lightbeam/about.html]: Same here missing `'`

**Comment by Lord-of-Darkness** [comment]
Desarrolladores y líderes de Mozilla, que son travestis culo!

Alguien debe colgar ustedes para arriba en sus propias bolas y dejar que su perro muerda su caricatura de una polla!

Posteriormente deben ponerlo tiempo que sea necesario por la garganta hasta que traga a sí mismos en el suicidio!

Ustedes Mozilla está ... Estúpido de mierda coños!
--------------------------------------------------------
Developers and leaders of Mozilla, which are shemales ass!

Someone should hang you up in your own balls and let your dog bite his caricature of a cock!

They should then put it long as needed by the throat until he swallows themselves in suicide!

You Mozilla is... Stupid fucking pussies!

**Comment by fmarier** [comment]
Added the missing quotes and indented the if/else/endif.

**Comment by alexgibson** [comment]
Thanks for the update @fmarier, r+! :racehorse:

**Pull request merged**